internal/idp/providers/ldap/session.go

```diff
@@ -51,8 +51,22 @@ func (s *Session) FetchUser(_ context.Context) (idp.User, error) {
 	searchRequest := ldap.NewSearchRequest(
 		s.Provider.baseDN,
 		ldap.ScopeWholeSubtree, ldap.NeverDerefAliases, 0, 0, false,
-		fmt.Sprintf("(&(objectClass="+s.Provider.userObjectClass+")("+s.Provider.userUniqueAttribute+"=%s))", ldap.EscapeFilter(s.user)),
-		[]string{"dn"},
+		fmt.Sprintf("(&(objectClass="+s.Provider.userObjectClass+")("+s.Provider.userUniqueAttribute+"=%s))", ldap.EscapeFilter(s.User)),
+		[]string{"dn",
+			s.Provider.idAttribute,
+			s.Provider.firstNameAttribute,
+			s.Provider.lastNameAttribute,
+			s.Provider.displayNameAttribute,
+			s.Provider.nickNameAttribute,
+			s.Provider.preferredUsernameAttribute,
+			s.Provider.emailAttribute,
+			s.Provider.emailVerifiedAttribute,
+			s.Provider.phoneAttribute,
+			s.Provider.phoneVerifiedAttribute,
+			s.Provider.preferredLanguageAttribute,
+			s.Provider.avatarURLAttribute,
+			s.Provider.profileAttribute,
```

Comment:
i think if ask before, but is there no problem if we send empty attributes?

Reply:
Empty attributes should not be a problem, but can handle them that they are not send unnecessarily